contrib/runners/python_runner/python_runner/python_runner.py

```diff
@@ -212,15 +226,19 @@ def run(self, action_parameters):
         command_string = list2cmdline(args)
         LOG.debug('Running command: PATH=%s PYTHONPATH=%s %s' % (env['PATH'], env['PYTHONPATH'],
```

Comment:
If stdin is used we should ideally print the whole command user could use to replicate, so something like `echo -e "serialized stdin params\n" | python ....`